src/app/modules/group/pages/group-user/group-user.component.html

```diff
@@ -21,5 +21,42 @@
       </div>
     </div>
 
+    <nav class="nav-tab" *ngIf="(currentUserGroupId$ | async) === user.groupId || isInitPagePersonal">
+      <a
+          class="nav-tab-item"
+          [routerLink]="'./'"
+          routerLinkActive #progress="routerLinkActive"
+          [routerLinkActiveOptions]="{exact: true}"
+          [class.active]="progress.isActive"
+          i18n
+      >
+        Progress
+      </a>
+      <a
+          class="nav-tab-item"
+          [routerLink]="'./personal-data'"
+          routerLinkActive #personalData="routerLinkActive"
+          [routerLinkActiveOptions]="{exact: true}"
+          [class.active]="personalData.isActive"
+          i18n
+      >
+        Personal data
+      </a>
+    </nav>
+
+    <ng-container *ngIf="progress?.isActive || !progress && (currentUserGroupId$ | async) !== user.groupId">
+      <p i18n>List progress</p>
+    </ng-container>
+
+    <ng-container *ngIf="personalData?.isActive">
+      <ng-container *ngIf="currentUserGroupId$ | async as currentUserGroupId">
+        <div class="bg-white" *ngIf="currentUserGroupId !== user.groupId">
+          <p class="no-access" i18n>You cannot access this page for this user</p>
+        </div>
+
+        <alg-current-user *ngIf="currentUserGroupId === user.groupId"></alg-current-user>
+      </ng-container>
```

Comment:
May be more readable:
```suggestion
      <ng-container *ngIf="(currentUserGroupId$ | async) === user.groupId; then currentUser; else forbidden "></ng-container>
      <ng-template #currentUser>
        <alg-current-user></alg-current-user>
      </ng-template>
      <ng-template #forbidden>
        <div class="bg-white">
            <p class="no-access" i18n>You cannot access this page for this user</p>
          </div>
      </ng-template>
```

Reply:
Fixed